diagnostic.md

```diff
@@ -4,7 +4,10 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
-
+h4.mb1
+a
+td.message
+div.filewrap
```

Comment:
not sure where you're looking for these, but I did not find any of these as selectors on the page

a few examples are `.text-gray`,  `a.user-mention`, `img.avatar`, `article`